src/app/Dashboard/styles/dashboard.scss

```diff
@@ -0,0 +1,146 @@
+.wrapper {
+  display: flex;
+  justify-content: space-around;
+}
+
+.main-content {
+  display: flex;
+  position: absolute;
+  top: 0;
+  z-index: 1;
+  width: 90%;
+}
+
+.sub-content {
+  flex-grow: 100;
+  font-family: 'DIN Pro';
+  padding-top: rem(23px);
+}
+
+.user-dashboard {
+  margin-top: rem(27px);
+}
+.user-dashboard__name {
+  @include text-styles(rem(27px), $lightAccent, 500, rem(0.56px), rem(27px));
+}
+
+.user-dashboard__level {
+  height: rem(23px);
+  width: rem(23px);
+  padding: rem(1px);
+  margin: rem(15px) 0;
+  @include text-styles( rem(14px), $orange, 500, rem(0.48px), rem(18px));
+  border: rem(1px) solid $orange;
+}
+
+.profile-overview {
+  display: flex;
+  justify-content: space-between;
+}
+
+
+.profile-overview__image {
+  height: 134px;
```

Comment:
I prefer use of percentages or ems vs pixels. I would like to hear your thoughts here.

Reply:
I missed out on converting this to rems. I have used rems but let me research on `ems`.